optaplanner-examples/src/main/java/org/optaplanner/examples/coachshuttlegathering/domain/Bus.java

```diff
@@ -79,6 +80,14 @@ public void setNextStop(BusStop nextStop) {
         this.nextStop = nextStop;
     }
 
+    public Integer getPassengerQuantityTotal() {
+        return passengerQuantityTotal == null ? 0 : passengerQuantityTotal;
+    }
+
+    public void setPassengerQuantityTotal(final Integer passengerQuantityTotal) {
```

Comment:
I'd be great if I could make this an `int`,  but Planner fail-fasts, saying that it can not be an `int`, as Planner needs null to be able to say whether it's initialized or not.

I think that that's non-sense in this case.